tests/core/identity/test_appkey.rs

```diff
@@ -0,0 +1,51 @@
+use ya_test_framework::framework::macros::{prepare_test_dir, serial_test};
+use ya_test_framework::framework::{framework_test, YagnaFramework};
+use ya_test_framework::utils::YagnaCli;
+
+#[cfg_attr(not(feature = "framework-test"), ignore)]
+#[framework_test]
```

Comment:
Suggestion: I would rename it to be more specific. `framework_test` seems too generic.
Proposals:
`yagna_test`
`ya_test`
Or to include "layer" of tests:
`e2e_test`
`system_test`

Reply:
I agree. We could split tests in multiple test sets, because execution will be rather long.
Maybe let's make survey on priv channels